io_scene_hubs/__init__.py

```diff
@@ -1,30 +1,123 @@
+import bpy
+
 from . import settings
 from . import components
 from . import operators
 from . import panels
+from .gather_properties import gather_properties
 
 bl_info = {
     "name" : "io_scene_hubs",
     "author" : "Robert Long",
     "description" : "",
-    "blender" : (2, 80, 0),
-    "version" : (0, 0, 1),
+    "blender" : (2, 82, 0),
+    "version" : (0, 0, 2),
```

Comment:
We should probably bake the exporter extension version into the gltf file. We can either append it to the `gltf.asset.generator` string or put it in `gltf.asset.extras.HUBS_gltfExporterVersion` or something like that.

Reply:
yeah that would be very useful